src/main/scala/streamit/implicits/implicits.scala

```diff
@@ -0,0 +1,87 @@
+package streamit
+
+import cats.effect.{ ConcurrentEffect, Timer }
+import cats.{ MonadError, Show }
+import fs2.{ Chunk, Pipe, Pull, Stream }
+import io.circe.parser.parse
+import io.circe.{ Json, JsonNumber, JsonObject }
+import streamit.util.CirceOpticsParser
+
+import scala.concurrent.duration.FiniteDuration
+import scala.language.implicitConversions
+
+package object implicits {
```

Comment:
it looks like that all the implicit here are ops for syntaxes. An option could be to use `syntax` as a name.

Reply:
Oh, that looks a lot less ugly, great suggestion.